packages/react/src/components/FileUploader/FileUploader.js

```diff
@@ -194,72 +195,56 @@ export class FileUploaderButton extends Component {
   }
 }
 
-export class Filename extends Component {
-  static propTypes = {
-    /**
-     * Specify an optional object of styles to be applied inline to the root
-     * node
-     */
-    style: PropTypes.object,
-
-    /**
-     * Specify the status of the File Upload
-     */
-    status: PropTypes.oneOf(['edit', 'complete', 'uploading']),
-
-    /**
-     * Provide a description for the complete/close icon that can be read by screen readers
-     */
-    iconDescription: PropTypes.string,
-  };
-
-  static defaultProps = {
-    onKeyDown: () => {},
-    status: 'uploading',
-    style: {},
-    tabIndex: 0,
-  };
-
-  render() {
-    const { iconDescription, status, style, ...other } = this.props;
-
-    if (status === 'uploading') {
+export function Filename({ iconDescription, status, ...other }) {
+  switch (status) {
+    case 'uploading':
+      return <InlineLoading iconDescription={iconDescription} />;
```

Comment:
Given there seems no `description` prop, any reason you chose `<InlineLoading>` over `<Loading>`?